ear/src/main/application/META-INF/was.policy

```diff
@@ -129,6 +129,7 @@ permission javax.management.MBeanPermission "*", "addNotificationListener";
 permission javax.management.MBeanPermission "*", "unregisterMBean";
 permission javax.management.MBeanPermission "*", "queryNames";
 permission java.lang.management.ManagementPermission "monitor";
+permission javax.management.MBeanPermission "-#-[-]", "queryMBeans";
```

Comment:
Heb je hier met reden `-#-[-]` gebruik in plaats van `*`?

Reply:
In de logging werd de melding met `-#-[-]` gegeven, dat is de reden dat ik die heb gebruikt.

Ik heb in een eerdere versie van de applicatie waarin het probleem optrad het wel met `*` geprobeerd, maar ik vermoed dat dat niet gewerkt heeft omdat ik het uiteindelijk toch naar  `-#-[-]` heb aangepast. Indien gewenst kan ik het nogmaals proberen met `*`.

Reply:
Graag nog een keer proberen met `"*"`, en als dat dan niet werkt een commentaar daarover toevoegen.